src/FrameworkDispatcher.cs

```diff
@@ -12,7 +12,6 @@
 
 using Microsoft.Xna.Framework.Audio;
 using Microsoft.Xna.Framework.Input.Touch;
-using Microsoft.Xna.Framework.Media;
```

Comment:
Does changing this to `using MediaPlayer = Microsoft.Xna.Framework.Media.MediaPlayer` fix it?

Reply:
yup!